test/e2e/tests/test_network/test_network.py

```diff
@@ -21,6 +21,10 @@
 ]
 ids = [sinsp.generate_id(sinsp_example) for sinsp_example in sinsp_examples]
 
+# For some reason, the modern probe gives a longer proc.exe than the legacy
+# drivers, needs further investigation.
```

Comment:
> * 1 test is failing because the test expects `proc.exe` to be `nginx: master proces`, but the modern probe sets it to `nginx: master process nginx -g daemon off;`. This is not a major problem IMO, but some extra investigation is needed to understand why this happens before changing the test to pass.

Checked w/ `bpf_printk` statements while running the e2e tests locally ... not a truncation error, this is all we got in the old eBPF raw_syscalls tracepoints for `argv[0]` aka `exe` ... @Andreagit97 would you know some good docs stating why? Clearly the `modern_bpf` driver has the complete arg. 

This demonstrates in an excellent way how much we need these e2e tests 🎉 .

Reply:
is there a way to reproduce this test locally by running it with just the modern bpf probe?